.tekton/open-infra-deployment-pr.yaml

```diff
@@ -39,6 +41,8 @@ spec:
             value: $(params.git-url)
           - name: REVISION
             value: $(params.revision)
+          - name: SCRIPT_IMAGE
```

Comment:
What is the purpose of that new param? I don't see it being used.

Reply:
> What is the purpose of that new param? I don't see it being used.

To get the build output in [deploy.yaml](https://github.com/redhat-appstudio/infra-deployments/pull/2383/files#diff-5a2a995c501aca7ed40fb6724a47d82f3f4a4db2cc8b4c3677cfffecfa39b507) file, we need `kustomize` cli but in the existing task [script image](https://github.com/redhat-appstudio/build-definitions/blob/main/task/update-infra-deployments/0.1/update-infra-deployments.yaml#L28), it does not have kustomize binary. Basically we need an image which has `kustomize` as well, also I don't want everyone to get affected with this image change.

> And by the way, somebody needs to minimally pull down quay.io/mkovarik/ose-cli-git:4.11 and store it in a quay repo other that Michal Kovarik's, who no longer is at Red Hat. What if he decides to delete his quay account?

+1 to this